api/pom.xml

```diff
@@ -1,14 +1,16 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.dragonet.proxy</groupId>
+        <artifactId>dragonproxy-parent</artifactId>
+        <version>0.3.4.3-SNAPSHOT</version>
+    </parent>
+
     <artifactId>dragonproxy-api</artifactId>
-    <version>0.0</version>
-    <name>DragonProxy-API</name>
-    <groupId>org.dragonet</groupId>
-    
-    <properties>
-        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-        <maven.compiler.source>1.8</maven.compiler.source>
```

Comment:
These properties are removed, but never replaced. They're especially important for synchronization between OSes (Mac, Linux, Windows), and for Java bytecode versioning.

Reply:
Encoding: https://github.com/sgdc3/DragonProxy/blob/master/pom.xml#L25
Compiler source version: https://github.com/sgdc3/DragonProxy/blob/master/pom.xml#L44

Reply:
My bad, didn't see these